plugins/modules/cloud/scaleway/scaleway_volume.py

```diff
@@ -106,11 +110,14 @@
 
 
 def core(module):
+    region = module.params["region"]
     state = module.params['state']
     name = module.params['name']
     organization = module.params['organization']
+    project = module.params['project']
     size = module.params['size']
     volume_type = module.params['volume_type']
+    module.params['api_url'] = SCALEWAY_LOCATION[region]["api_endpoint"]
```

Comment:
Is this a bugfix? Is it related to the other bugfixes? If not, it should be a separate PR.

Reply:
I need this part to have the scw aliases. example par1 = fr-par-1